pkgs/development/libraries/libgcrypt/default.nix

```diff
@@ -36,8 +38,11 @@ stdenv.mkDerivation rec {
     mkdir -p $out/lib
     cp src/.libs/libgcrypt.20.dylib $out/lib
   '';
+  configureFlags = [
+    ''--with-libgpg-error-prefix=${libgpgerror.dev.outPath}''
+  ];
```

Comment:
I am not sure this works at all. `libgcrypt`/`libgpgerror` produce some asm code during configure by producing binaries according to the machine architecture. I know these issues from my experiments with webassembly, which is similar to cross-compilation.

Reply:
I had a look at the git repo of libgcrypt and i meant [libgcrypt/mpi](https://git.gnupg.org/cgi-bin/gitweb.cgi?p=libgcrypt.git;a=tree;f=mpi;h=17a9cb267cbfa8e7f2e2ba7f1fe8f1b8f767a78e;hb=HEAD) in my previous comment.